lib/enum/nested.rb

```diff
@@ -18,21 +18,27 @@ def initialize(people_array)
   end
 
   def ages_sum
+    people.map{|person| person.age}.inject(:+)
   end
 
   def average_age
+    people.ages_sum / people.length
   end
 
   def total_years_programming_experience_for_all_languages
+    people.map{|person| person.years_language_experience.values}.flatten.inject(:+)
   end
 
   def favorite_food_frequency
+    people.map {|person| person.favorite_foods}.flatten.group_by{|i| i}.
```

Comment:
You've got a syntax error here, with the `.` at the end of line 33. That's why your tests aren't running.